datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Domain/SetDomainModal.tsx

```diff
@@ -48,35 +42,53 @@ export const SetDomainModal = ({ visible, onClose, refetch }: Props) => {
 
     const inputEl = useRef(null);
 
-    const onOk = async () => {
-        if (!selectedDomain) {
-            return;
-        }
-        try {
-            await setDomainMutation({
+    useEffect(() => {
+        setTimeout(() => {
+            if (inputEl && inputEl.current) {
+                (inputEl.current as any).focus();
+            }
+        }, 1);
+    });
+
+    const recommendedDomainsData = GetDomainRecommendation();
```

Comment:
Let's rename from GetDomainRecommendation to

useGetRecommendedDomains

Reply:
Checked and fixed.